public/js/components/SourcesTree.js

```diff
@@ -1,18 +1,16 @@
 const React = require("react");
 const { DOM: dom, PropTypes } = React;
+const classnames = require("classnames");
+const ImPropTypes = require("react-immutable-proptypes");
+const { Set } = require("immutable");
 
 const {
   nodeHasChildren, createParentMap, addToTree,
   collapseTree, createTree
 } = require("../utils/sources-tree.js");
-
-const classnames = require("classnames");
-const ImPropTypes = require("react-immutable-proptypes");
-const { Set } = require("immutable");
-const debounce = require("lodash/debounce");
-
 const ManagedTree = React.createFactory(require("./utils/ManagedTree"));
 const Svg = require("./utils/Svg");
+const { throttle } = require("../utils/utils");
```

Comment:
Lodash uses `throttle` slightly differently https://lodash.com/docs#throttle

Do you think we could call this `delay`, `queue`, `defer`, or `batch`